ros_cross_compile/sysroot_creator.py

```diff
@@ -111,6 +113,14 @@ def prepare_docker_build_environment(
     _copy_or_touch(custom_setup_script, docker_build_dir / 'user-custom-setup')
     _copy_or_touch(custom_post_build_script, docker_build_dir / 'user-custom-post-build')
 
+    if colcon_defaults_file:
```

Comment:
I am only just seeing now that this broke the ability to use a `defaults.yaml` file placed in the workspace. If the user does not explicitly state the `--colcon-defaults` argument, then this default version will be written, there is no checking for a file.